hashintel/hash
# Implement BlockHighlight to highlight linked block

## 🌟 What is the purpose of this PR?

- Highlights the block by rendering `BlockHighlight` on the left side of the block.
- Only highlights the block once, when the page is loaded
- Determines which block to highlight by the `routeHash` on the URL


## 🔗 Related links
[Asana task](https://app.asana.com/0/1200211978612931/1202689355023866/f) (internal)


## ❓ How to test this?

1.  Checkout the branch / view the deployment
1.  Copy link of a block inside a page
1.  Open the copied link in a new tab
2. The copied block should be highlighted for 4seconds 

## 📹 Demo

https://user-images.githubusercontent.com/39553853/183899680-8cf58a92-4a6d-4887-be11-e9c45df0a8e3.mov
